pkgs/development/tools/ocaml/opam/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, lib, fetchurl, makeWrapper, getconf,
-  ocaml, unzip, ncurses, curl, bubblewrap
+  ocaml, unzip, ncurses, curl, bubblewrap, Foundation
```

Comment:
```suggestion
  ocaml, unzip, ncurses, curl, bubblewrap, CoreServices, Foundation
```

Reply:
@Et7f3 There is an inheritance link [Foundation -> ApplicationServices -> CoreServices](https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix), so we don't need to add CoreServices explicitly.

Reply:
I did a PR upstream that change Foundation to CoreFoundation so we will need to reference CoreServices in next version. Also doesn't the goal of nix is to provide explicit list ? Because CoreService was referenced explicitly in the build.

Reply:
> I did a PR upstream that change Foundation to CoreFoundation so we will need to reference CoreServices in next version.

Then we can do this in the next version.

> Also doesn't the goal of nix is to provide explicit list ? Because CoreService was referenced explicitly in the build.

Making dependencies explicit is usually a good thing, but it is unneeded, as in this case, if CoreServices is a propagated dependency of Foundation.